PhotoFeed/app/src/main/java/com/mobileproto/jovanduy/photofeed/DbAccessor.java

```diff
@@ -0,0 +1,98 @@
+package com.mobileproto.jovanduy.photofeed;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.util.Log;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * Class to modify the database
+ */
+public class DbAccessor {
```

Comment:
I think in my mind, DbAccessor was a class that would be instantiated and used by the fragment classes, while FeedReaderDbHelper was more or less a class of constants that would not be directly accessed by the fragment classes, but rather just by DbAccessor. Looking back on it, I guess I could've combined the two classes.